File: mobile-client/android/Contract/Message/Thread/MessageThreadSummary.kt
package app.mobiling.client.client.contract.message.thread

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class MessageThreadSummary(
    val threadId: String,
    val subject: String?,
    val lastMessagePreview: String,
    val unreadCount: Int,
    val updatedAtIso8601: String,
)

File: client/ios/Contract/Message/Thread/MessageThreadSummary.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct MessageThreadSummary: Sendable, Codable {
    public let threadId: String
    public let subject: String?
    public let lastMessagePreview: String
    public let unreadCount: Int
    public let updatedAtIso8601: String
}
